### DISCUSSION TITLE
Pelt convergence for large penalties

### DISCUSSION THREAD

**Discussion by JB-Git-15** [deepcharles/ruptures#286] [General]
Hi 
  I'm trying Pelt, and I see that the computation time increases a lot when using large penalties.
 Is it possible to modify some tolerance parameter or the number of iterations ? 
Best regards 

JB

**Comment by deepcharles** [comment]
Hi, if you use `rpt.Pelt`, you can change the `jump` parameter as follows: `algo = rpt.Pelt(jump=k)`. It means that only multiple of k are considered when looking for change point indexes (k, 2k, 3k, etc.). By default, `k` is set to 5 and if you increase it (to, say, 10) the algorithm should be faster.
